consensus/polybft/consensus_runtime.go

```diff
@@ -210,6 +218,39 @@ func (c *consensusRuntime) initCheckpointManager(logger hcf.Logger) error {
 	return nil
 }
 
+// initBurntFeesManager initializes burnts fees manager
+// if bridge is not enabled, then a dummy burnt fees manager will be used
+func (c *consensusRuntime) initBurntFeesManager(logger hcf.Logger) error {
```

Comment:
Same here with the logger.

Reply:
No idea, just following the current pattern. We could create a ticket to clean up this part of the code.